src/main/webapp/i18n/de/exam.json

```diff
@@ -37,7 +37,7 @@
             "endExam": "Beenden",
             "date": "Datum",
             "time": "Zeit",
-            "durationWithFormat": "Dauer (mm:ss)",
+            "durationWithFormat": "Dauer",
```

Comment:
The `durationWithFormat` translation is now the same as the `exam.overview.duration`, I would suggest removing the first one and exchange it with the second translation that could be moved up one level.